tests/api-watch/test_diff_module.ml

```diff
@@ -118,8 +119,7 @@ let%expect_test "Submodules with different functor types" =
   Format.printf "%a" pp_diff_option result;
   [%expect
     {|
-    Some (Module Main: {Modified (Supported [ Module F: {Modified (Unsupported)};
```

Comment:
Why did this change? There is no apparent reasons why we couldn't report this change anymore!

Reply:
Sorry my bad, we do support functors, but if there was a change was just report an unsupported change on them. It seems that `Includemod.modtypes` doesn't detect a change on functor parameters anymore for some reason. I need to investigate this further.